_posts/en/newsletters/2018-11-20-newsletter.md

```diff
@@ -0,0 +1,341 @@
+---
+title: "Bitcoin Optech Newsletter #22"
+permalink: /en/newsletters/2018/11/20/
+name: 2018-11-20-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's "wumbo-sized" newsletter provides a note about Bitcoin hash
+rate related to forks on other coins, summarizes several accepted goals
+for the Lightning Network protocol version 1.1 specification, and lists
+several notable commits in popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Monitor feerates:** due to activity associated with Bitcoin Cash,
+  which can use the same mining equipment used for Bitcoin, blocks on
+  Bitcoin may appear less frequently than expected, raising feerates and
+  causing other effects.  However, these conditions may suddenly
+  reverse, providing a period of fast blocks and low fee rates.  See the
+  News section below for more information and recommended actions
+  for Bitcoin businesses.
+
+## Feature News: Lightning Network protocol 1.1 goals
+
+LN protocol developers [met in Adelaide][rusty wrapup] last weekend to
+determine which changes to adopt for the forthcoming Lightning Protocol
+Specification 1.1.  Thirty proposals were accepted at a
+high-level---meaning full specifications for each proposal are not
+necessarily defined or agreed upon yet---but the basic outline of the
+new features is available.  Below, Optech has attempted to briefly
+describe the highlights from the meeting mentioned in an email linking
+to the [working outline][ln1.1 outline] by Rusty Russell: "multi-path
+payments, dual-funded channels, splicing, wumbo, hidden destinations,
+and many gossip improvements."
+
+- **Multi-path payments:** the current normal way to make a payment over
+  LN is using a single path.  Alice pays Charlie through her channel to
+  Bob and Bob's channel to Charlie.  This works well for small payments
+  where each participant has enough balance or capacity in their channel
+  to support the payment.  But if we use this mechanism when Alice has
+  10 open channels each containing a maximum of 10% of her total hot
+  wallet balance, Alice can only spend at most 10% of her funds at a
+  time.
+
+    Multipath payments provide a solution to this problem: if Alice
+    wants to send 15% of her funds, she can send 7.5% to Charlie through
+    her channel with Bob and 7.5% through her channel with Dan (who also
+    has a channel with Charlie).  Although the partial payments are
+    routed through separate paths, they can each commit to the same
+    hash Alice would've used to send a single-path payment.  If Charlie
+    receives multiple payments within a reasonable time period that
+    equal or exceed the expected amount, he can reveal their shared
+    preimage to claim them simultaneously---providing the same security
+    currently used for single-path payments.  This also works if some
```

Comment:
A new paragraph!  And ruin my precious parallelism between each of these points!  Are you crazy!  :-)

Seriously, I'm confused by your remark.  Splitting further along the path was something I deliberately didn't describe because the [base AMP proposal](https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-November/001577.html) indicated it might be unsafe so I don't want to promise it.  What's described here is merging along the path---to make that clearer, I'm revising the phrasing slightly.

I don't think a single sentence about path merging is enough to warrant a new paragraph, so I'm leaving the sentence where it is (feel free to overrule), but if you want an upgrade to an additional sentence about non-spender path splitting, I'll throw in the extra paragraph for free.  :-)

Reply:
I think it looks great as it is in your latest commit.

I hadn't read the Base AMP emails before my comment and was going off my (limited) knowledge of OG AMP where I thought arbitrary splits were allowed and payment was guaranteed to be atomic.

I don't understand the point in zmn's email about the safety of arbitrary splits. Hopefully at some point there'll be an explanation on the mailing list of why that is.